server/container/guice/protocols/managedsieve/src/main/java/org/apache/james/modules/protocols/SieveProbeImpl.java

```diff
@@ -68,9 +72,16 @@ public void removeSieveQuota(String user) throws Exception {
     }
 
     @Override
-    public void addActiveSieveScript(String username, String name, String script) throws Exception {
-        User user = User.fromUsername(username);
+    public void addActiveSieveScript(String userName, String name, String script) throws Exception {
+        User user = User.fromUsername(userName);
         sieveRepository.putScript(user, new ScriptName(name), new ScriptContent(script));
         sieveRepository.setActive(user, new ScriptName(name));
     }
+
+    @Override
+    public void addActiveSieveScriptFromFile(String userName, String name, String path) throws Exception {
+        try (InputStream scriptFileAsStream = new FileInputStream(path)) {
+            addActiveSieveScript(userName, name, IOUtils.toString(scriptFileAsStream));
```

Comment:
`IOUtils.toString(InputStream)` is deprecated.

We need to specify the encoding.

We generally assume that `StandardCharset.UTF-8` does the job.